### PR TITLE
support easily getting into mysql

### PR DESCRIPTION
**Pull request by mulby**
Convenience target for dropping you into a mysql CLI

@edx/docker-devstack-working-group

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -95,3 +95,6 @@ validate-lms-volume: ## Validate that changes to the local workspace are reflect
 	touch $(DEVSTACK_WORKSPACE)/edx-platform/testfile
 	docker exec edx.devstack.lms ls /edx/app/edxapp/edx-platform/testfile
 	rm $(DEVSTACK_WORKSPACE)/edx-platform/testfile
+
+dbshell:
```
Comment (clintonb) [inline Makefile]:
```
mysql-shell:
  docker-compose exec edx.devstack.mysql bash
`

**Review by clintonb — COMMENTED** [review]
I'm on the fence regarding whether to drop users into the system shell or MySQL.

File: Makefile (hunk 1 above)
Comment (mulby) [reply Makefile]: Good suggestion, done.

File: Makefile [hunk 2]
```diff
@@ -95,3 +95,6 @@ validate-lms-volume: ## Validate that changes to the local workspace are reflect
 	touch $(DEVSTACK_WORKSPACE)/edx-platform/testfile
 	docker exec edx.devstack.lms ls /edx/app/edxapp/edx-platform/testfile
 	rm $(DEVSTACK_WORKSPACE)/edx-platform/testfile
+
+mysql-shell:
```
Comment (clintonb) [inline Makefile]: Add help text: `## Run a shell on the mysql container`

**Review by clintonb — APPROVED** [review]
Remember to squash your commits.

**Pull request merged**